core/src/main/python/encrypt.py

```diff
@@ -253,4 +253,7 @@ def main(args):
 
 if __name__ == '__main__' or __name__ == 'main':
     WebLogicDeployToolingVersion.logVersionInfo(_program_name)
-    main(sys.argv)
+    try:
+        main(sys.argv)
+    except:
```

Comment:
The encryptModel unit test is failing because this `except` is catching Jython `exceptions.SystemExit` exception.  Several non-WLST tools call `sys.exit()` directly and I assume would have the same failure. Other tools call the underlying Java `System.exit()`, which doesn't seem to trigger that exception. I'm not sure if this code should be modified to let that exception pass, or if those `sys.exit()` calls should change to Java `System.exit()`.

Reply:
Thanks for letting me know.  Still exploring options, as you can see in my reply to Robert's comment.  I'm all ears.

Reply:
If sys.exit() is throwing a Jython exception, my feeling is that we should switch to System.exit()